### ISSUE TITLE
Problem with setting Custom Prefix with Colors (red from permission)

### ISSUE THREAD

**Issue by se7enek** [Proxying/TabManager#22]
Inserting following code:
`      "prefix": "%player_prefix% %",
      "suffix": "% --- &aColored&r _notcolored_ {notcolored %notcolored {colored} %colored%"`

produces that result:
http://se7en.pl/upload/1558000067.png

Expected behaviour? No {} visible around the name of player AND colored nickname.

It's reading colors from LuckPerms properly:
`  meta:
  - namecolor:
      value: red`

Any way to achieve that, aka im doing it wrong, or just not possible?
P.s. My prefix is different for each of the "Player Title", not related to color groups.

**Comment by se7enek** [comment]
Creating new Meta for simple "color", and using `&%player_option_color%` simply produces "&2" in tab.
Expected colored.

**Comment by se7enek** [comment]
@Proxying

**Comment by Proxying** [comment]
%placeholder% are handled by PlaceholderAPI. Does this have the same issue/functionality with other plugins that implement PAPI? For example a scoreboard plugin?

**Comment by se7enek** [comment]
This is the scoreboard plugin... I dont use any else. But in other plugins when i use &%player_option_color%Text, it is colored. Any way to make your plugin (tabmanager) to actually read same exact format from Chat? (Using Nucleus+LuckPerms)

**Comment by Proxying** [comment]
TabManager doesn’t handle scoreboards, only tab menu. I don’t think theres a way to replicate chat, however you can hard code the group colours, or add individual colours to players.json

**Comment by se7enek** [comment]
Yeah but the main problem is that i want my "Tab" to use data from 2 groups at same time.
I got one group like unregistered/registered/vip/mod/admin (recognizable by name's color) + i got 2nd group that is playtime of player ie. "Newbie" "Casual Player" "Long Player" "Im here over a year" etc.

I want both data to be displayed in Tab in that fashion: 
`[Newbie] <color=red>NICKNAME</color>`

Is it possible other way?

**Comment by se7enek** [comment]
It would be actually nice if you could just allow something like "Global" format, without group requirement, so others could make it just like "Format=%prefix %displayname %suffix", just like it is in any other chat management plugin. Im not programmer so no idea if possible, but should be?
Previous question stays.

@Proxying

**Comment by se7enek** [comment]
Downloaded YYSSB scoreboard plugin and when i used prefix="%player_prefix% &e" it displayed prefix properly. Back to your plugin, when i use %player_prefix%, it shows prefix same for every player (equal to prefix of the one pressing TAB).

Any solution for that? 
@Proxying

**Comment by Proxying** [comment]
https://github.com/Proxying/TabManager/issues/17

**Comment by se7enek** [comment]
I see some common things in both issues, yet, other plugins like YYSSB scoreboard, which is using PlaceholderAPI too, is showing stuff correct. Thats why i can tell that its not something with PAPI, but with TabManager.

If plugin A uses plugin X and it does work, how fact that plugin B using same plugin X isnt working leads you to conclusion that plugin X is broken?

**Comment by Proxying** [comment]
I don't know what you're talking about. I referenced that issue because the feature you are asking for has already been asked for in that issue. And I've responded to that request.
